components/advanced_text_editor/advanced_text_editor.scss

```diff
@@ -171,6 +179,18 @@
                 border-left: none;
             }
         }
+
+        .ThreadViewer & {
+            .msg-typing:empty {
```

Comment:
this is fine, but keep in mind that this selector is super flaky regarding white-spaces and inner elements, even if they are not showing/displaying or have no content on their own

Reply:
@michelengelen are you referring to empty pseudo selector.

Reply:
yes, I am referring to that ... I had problems with that in the past, but when we are careful with what we insert in that `.msg-typing` element we are good to go!